metricbeat/docs/modules/docker.asciidoc

```diff
@@ -33,6 +33,9 @@ metricbeat.modules:
   hosts: ["unix:///var/run/docker.sock"]
   period: 10s
 
+  # Replace dots in labels with `_`. Set to false to keep dots
+  dedot: true
```

Comment:
Can we call this `labels.dedot` to make it clear it's not dedotting the complete event?